analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -177,7 +177,11 @@ open class Npm(
             licenseNode["type"]?.textValue()
         }
 
-        return declaredLicenses
+        return declaredLicenses.map {
+            // NPM does not mean https://unlicense.org/ here, but the wish to not "grant others the right to use a
+            // private or unpublished package under any terms", which corresponds on SPDX' "NONE".
```

Comment:
Should be "SPDX's" (had to check myself: https://ell.stackexchange.com/questions/6295/when-a-word-ends-in-s-or-x-do-you-add-s-or-just-an). Also, "corresponds to".

Reply:
Done.